app/frontend/styles/_provider_header.scss

```diff
@@ -5,7 +5,13 @@
 }
 
 .provider-header__content {
+  .govuk-header__navigation { float: right }
+
   @include govuk-media-query($from: tablet) {
     width: 40%;
   }
+
+  @include govuk-media-query($until: desktop) {
+    .govuk-header__navigation { float: left }
```

Comment:
Or this

Reply:
Without this it floats left on small screens - if this is okay we don't need this CSS:

<img width="993" alt="Screenshot 2020-06-12 at 11 37 35" src="https://user-images.githubusercontent.com/13377553/84494287-20e7dd80-aca1-11ea-8ca6-845b71d43854.png">